eng/pipelines/scripts/mgmt_sdk_compiler.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+#args expected
+# $1. Java version : 1.7 or 1,8, defaults to "1.8"
+# $2. Goals, defaults to "clean compile"
+
+echo "#### CWD : "
+pwd
+
+echo "#### versions of java available:" 
+ls /usr/lib/jvm
+
+echo "#### Original java home $JAVA_HOME"
+
+JAVA7HOME="/usr/lib/jvm/zulu-7-azure-amd64"
+JAVA8HOME="/usr/lib/jvm/zulu-8-azure-amd64"
+
+JAVAHOME="$JAVA8HOME"
+
+MAVENGOALS="clean compile"
+
+if [ -z "$1" ] && [ "$1" == "1.7" ];
+  then JAVAHOME="$JAVA7HOME"; echo "runing java 7 build";
+fi
+
+
+if [ -z "$2" ];
+  then MAVENGOALS="$2"; echo "maven goals overriden to $2"
```

Comment:
This is a step closer but wasn't fully what I was expecting. As we chatted as long as this works for you now it is fine and we can refine it overtime.